<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewUsers</class>
 <widget class="QDialog" name="ViewUsers">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1054</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #cddce8;</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1031</width>
     <height>611</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_10">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget_2">
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,0">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SizeConstraint::SetMinAndMaxSize</enum>
        </property>
        <item>
         <widget class="QListWidget" name="listWidget">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:#ffffff</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::Shape::NoFrame</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Viewmore_button">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background-color: #598bb8;   /* dark blue */
    color: white;
    border: none;                /* no border line */
    border-radius: 12px;         /* rounded corners */
    padding: 8px 16px;
    font-size: 19px;
    background-image: none;      /* no image */
}

QPushButton:hover {
    background-color: #0D1B5A;   /* darker on hover */
}</string>
          </property>
          <property name="text">
           <string>ViewMore</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QPushButton" name="backButton_2">
            <property name="font">
             <font>
              <family>Arial Unicode MS</family>
              <pointsize>-1</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
    background-color: #598bb8;   /* dark blue */
    color: white;
    border: none;                /* no border line */
    border-radius: 12px;         /* rounded corners */
    padding: 8px 16px;
    font-size: 19px;
    background-image: none;      /* no image */
}

QPushButton:hover {
    background-color: #0D1B5A;   /* darker on hover */
}</string>
            </property>
            <property name="text">
             <string>&lt;  BACK</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Policy::Preferred</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>200</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SizeConstraint::SetMinAndMaxSize</enum>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="username_label">
            <property name="font">
             <font>
              <family>Arial Unicode MS</family>
              <pointsize>22</pointsize>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>username</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="line_4">
          <property name="styleSheet">
           <string notr="true">border-top: 2px solid #598bb8;</string>
          </property>
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="info_label">
            <property name="font">
             <font>
              <family>Arial Unicode MS</family>
              <pointsize>18</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Information</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="line_3">
          <property name="styleSheet">
           <string notr="true">border-top: 2px solid #598bb8;</string>
          </property>
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_5">
              <property name="font">
               <font>
                <family>Arial Unicode MS</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>First Name:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="fname_label">
              <property name="font">
               <font>
                <family>Arial Unicode MS</family>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="frameShape">
               <enum>QFrame::Shape::NoFrame</enum>
              </property>
              <property name="text">
               <string>fname</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_8">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_6">
              <property name="font">
               <font>
                <family>Arial Unicode MS</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Last Name :</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lname_label">
              <property name="font">
               <font>
                <family>Arial Unicode MS</family>
                <pointsize>16</pointsize>
               </font>
              </property>
              <property name="frameShape">
               <enum>QFrame::Shape::NoFrame</enum>
              </property>
              <property name="text">
               <string>lname</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_9">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_7">
              <property name="font">
               <font>
                <family>Arial Unicode MS</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>National ID:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="id_label">
              <property name="font">
               <font>
                <family>Arial Unicode MS</family>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="frameShape">
               <enum>QFrame::Shape::NoFrame</enum>
              </property>
              <property name="text">
               <string>id</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="line_2">
          <property name="styleSheet">
           <string notr="true">border-top: 2px solid #598bb8;</string>
          </property>
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="sizeConstraint">
           <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
          </property>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="Email_label">
              <property name="font">
               <font>
                <family>Arial Unicode MS</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Email:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="email_lineedit">
              <property name="font">
               <font>
                <family>Arial Unicode MS</family>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color:#ffffff</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="frame">
               <bool>false</bool>
              </property>
              <property name="placeholderText">
               <string>Email</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <property name="spacing">
             <number>0</number>
            </property>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="font">
               <font>
                <family>Arial Unicode MS</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Phone Number :</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="phonenum_label">
              <property name="font">
               <font>
                <family>Arial Unicode MS</family>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="autoFillBackground">
               <bool>false</bool>
              </property>
              <property name="frameShape">
               <enum>QFrame::Shape::NoFrame</enum>
              </property>
              <property name="text">
               <string>phone number</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Policy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_4">
              <property name="font">
               <font>
                <family>Arial Unicode MS</family>
                <pointsize>16</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">border-top: 2px solid #598bb8;</string>
              </property>
              <property name="text">
               <string>Balance:</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <property name="spacing">
               <number>0</number>
              </property>
              <item>
               <widget class="QLineEdit" name="Balance_lineEdit">
                <property name="font">
                 <font>
                  <family>Arial Unicode MS</family>
                  <pointsize>14</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">background-color:#ffffff</string>
                </property>
                <property name="frame">
                 <bool>false</bool>
                </property>
                <property name="placeholderText">
                 <string>Balance</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_3">
                <property name="font">
                 <font>
                  <family>Arial Unicode MS</family>
                  <pointsize>14</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>EGP</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Orientation::Horizontal</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::Policy::Fixed</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>620</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <widget class="Line" name="line_5">
              <property name="styleSheet">
               <string notr="true">border-top: 2px solid #598bb8;</string>
              </property>
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_4">
              <property name="orientation">
               <enum>Qt::Orientation::Vertical</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Policy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <property name="spacing">
               <number>0</number>
              </property>
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Orientation::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="save_pushButton">
                <property name="styleSheet">
                 <string notr="true">QPushButton {
    background-color: #598bb8;   /* dark blue */
    color: white;
    border: none;                /* no border line */
    border-radius: 12px;         /* rounded corners */
    padding: 8px 16px;
    font-size: 19px;
    background-image: none;      /* no image */
}

QPushButton:hover {
    background-color: #0D1B5A;   /* darker on hover */
}</string>
                </property>
                <property name="text">
                 <string>Save</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Orientation::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer_3">
              <property name="orientation">
               <enum>Qt::Orientation::Vertical</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Policy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_8">
              <property name="spacing">
               <number>10</number>
              </property>
              <item>
               <widget class="QPushButton" name="edit_pushButton">
                <property name="styleSheet">
                 <string notr="true">QPushButton {
    background-color: #598bb8;   /* dark blue */
    color: white;
    border: none;                /* no border line */
    border-radius: 12px;         /* rounded corners */
    padding: 8px 16px;
    font-size: 19px;
    background-image: none;      /* no image */
}

QPushButton:hover {
    background-color: #0D1B5A;   /* darker on hover */
}</string>
                </property>
                <property name="text">
                 <string>Edit</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="ChangePassword">
                <property name="styleSheet">
                 <string notr="true">QPushButton {
    background-color: #598bb8;   /* dark blue */
    color: white;
    border: none;                /* no border line */
    border-radius: 12px;         /* rounded corners */
    padding: 8px 16px;
    font-size: 19px;
    background-image: none;      /* no image */
}

QPushButton:hover {
    background-color: #0D1B5A;   /* darker on hover */
}</string>
                </property>
                <property name="text">
                 <string>Change Password</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="sus_pushButton">
                <property name="styleSheet">
                 <string notr="true">QPushButton {
    background-color: #598bb8;   /* dark blue */
    color: white;
    border: none;                /* no border line */
    border-radius: 12px;         /* rounded corners */
    padding: 8px 16px;
    font-size: 19px;
    background-image: none;      /* no image */
}

QPushButton:hover {
    background-color: #0D1B5A;   /* darker on hover */
}</string>
                </property>
                <property name="text">
                 <string>Suspend</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="del_pushButton">
                <property name="styleSheet">
                 <string notr="true">QPushButton {
    background-color: #598bb8;   /* dark blue */
    color: white;
    border: none;                /* no border line */
    border-radius: 12px;         /* rounded corners */
    padding: 8px 16px;
    font-size: 19px;
    background-image: none;      /* no image */
}

QPushButton:hover {
    background-color: #0D1B5A;   /* darker on hover */
}</string>
                </property>
                <property name="text">
                 <string>Delete</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="vth_pushButton_2">
                <property name="styleSheet">
                 <string notr="true">QPushButton {
    background-color: #598bb8;   /* dark blue */
    color: white;
    border: none;                /* no border line */
    border-radius: 12px;         /* rounded corners */
    padding: 8px 16px;
    font-size: 19px;
    background-image: none;      /* no image */
}

QPushButton:hover {
    background-color: #0D1B5A;   /* darker on hover */
}</string>
                </property>
                <property name="text">
                 <string>View Transaction History</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Orientation::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
